NetBarcode/Barcode.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace NetBarcode
 {
-    public enum Type
+    public enum BarcodeType
```

Comment:
this change breaks compatibility. I will consider this suggestion for version 2. Revert all BarcodeType changes to Type
```suggestion
    public enum Type
```